### PR TITLE
Update documentation on how to use quotation marks in metadata

### PR DESCRIPTION
**Pull request by mmitoraj**
Changes proposed in this pull request:

- Updated documentation on how to use quotation marks in metadata
- Added quotation marks to the metadata of a troubleshooting document

**Related issue(s)**
See also #4532

### CODE REVIEW

File: docs/headless-cms/03-03-markdown-documents.md [hunk 1]
```diff
@@ -29,7 +29,11 @@ type: {Document type}
 ---
 ```
 
-- The `title` metadata defines the title of your document.
+- The `title` metadata defines the title of your document. If you want the title or its part to display inside quotation marks, place it between additional, single quotation marks or use a backslash (**/**) before the quoted word. See the following examples:
```
Comment (kazydek) [inline docs/headless-cms/03-03-markdown-documents.md]:
```suggestion
- The `title` metadata defines the title of your document. If you want the title or its part to render inside quotation marks, place it between additional, single quotation marks or use a backslash (**/**) before the quoted word. See the following examples:
```

File: docs/headless-cms/03-03-markdown-documents.md [hunk 2]
```diff
@@ -29,7 +29,11 @@ type: {Document type}
...
+  - ✅ `title: '"Transport is closing" error'`
```
Comment (kazydek) [inline docs/headless-cms/03-03-markdown-documents.md]:
```suggestion
 ✅ `title: '"Transport is closing" error'`
```

File: docs/headless-cms/03-03-markdown-documents.md [hunk 3]
```diff
@@ -29,7 +29,11 @@ type: {Document type}
...
+  - ✅ `title: \"Transport is closing" error`
```
Comment (kazydek) [inline docs/headless-cms/03-03-markdown-documents.md]:
```suggestion
 ✅ `title: \"Transport is closing" error`
```

File: docs/headless-cms/03-03-markdown-documents.md [hunk 4]
```diff
@@ -29,7 +29,11 @@ type: {Document type}
...
+  - ⛔️ `title: "Transport is closing" error`
```
Comment (kazydek) [inline docs/headless-cms/03-03-markdown-documents.md]:
```suggestion
⛔️ `title: "Transport is closing" error`
```

File: docs/headless-cms/03-03-markdown-documents.md [hunk 5]
```diff
@@ -29,7 +29,12 @@ type: {Document type}
 ---
 ```
 
-- The `title` metadata defines the title of your document.
+- The `title` metadata defines the title of your document. If you want the title or its part to render inside quotation marks, place it between additional, single quotation marks or use a backslash (**/**) before the quoted word. See the following examples:
+
+  ✅ `title: '"Transport is closing" error'`<br>
+  ✅ `title: \"Transport is closing" error`<br>
```
Comment (derberg) [inline docs/headless-cms/03-03-markdown-documents.md]: it should be `title: \"Transport is closing\" error`

http://docs.octoprint.org/en/master/configuration/yaml.html
```
In double quoted strings if you need to include a literal double quote in your string you can escape it by prefixing it with a backslash \ (which you can in turn escape by itself). In single quoted strings the single quote character can be escaped by prefixing it with another single quote, basically doubling it. Backslashes in single quoted strings do not need to be escaped.
```

**Pull request merged**